src/main/java/io/apimatic/coreinterfaces/type/functional/ContextInitializer.java

```diff
@@ -0,0 +1,22 @@
+package io.apimatic.coreinterfaces.type.functional;
+
+import java.io.IOException;
+import io.apimatic.coreinterfaces.http.Context;
+
+
+/**
+* Functional Interface to apply the context initialization function.
```

Comment:
update the description with the following,
Functional Interface to apply the context initialization function for the response models.

Reply:
Updated